src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -444,7 +445,7 @@ public ItemStack decrStackSize(final int index, final int count)
             tempIndex -= nonnulllist.size();
         }
 
-        return list != null && !(list.get(tempIndex)).isEmpty() ? ItemStackHelper.getAndSplit(list, tempIndex, count) : ItemStack.EMPTY;
+        return list != null && !(list.get(tempIndex)).isEmpty() ? ItemStackHelper.getAndSplit(list, tempIndex, count) : ItemStackUtils.EMPTY;
```

Comment:
Here is still an isEmpty() check.
Please replace that with the ItemStackUtil one.